### PR TITLE
Select member roles

### LINKED ISSUE #692: Implement Roles select
Implement the [Roles select](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=2985%3A111662):

*   It should allow selecting multiple roles
*   The selection should be confirmed by Apply Filter button
*   The selection can be cleared by Clear filter button
*   The selected roles should be displayed [as role icons](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=2993%3A2794)
    *   If there are more roles selected than we can display show (+NN)
    *   It looks like member roles component could be potentially re-used :thinking:

### LINKED ISSUE #828: Add a test suite for `common/utils` functions

### PR DESCRIPTION
**Pull request by thesan**
Closes  #692
Closes #828

### CODE REVIEW

**Comment by thesan** [comment]
It looks like #817 breaks the member filters box layout

File: packages/ui/src/common/components/selects/SimpleSelect.tsx [hunk 1]
```diff
@@ -188,9 +188,12 @@ const Options = styled.div`
   top: 100%;
   user-select: none;
   min-width: 100%;
-  z-index: 10;
+  z-index: 51;
```
Comment (jodator) [inline packages/ui/src/common/components/selects/SimpleSelect.tsx:191]: Let's put that to constants/styles.ts -> ZIndex variable

File: packages/ui/src/common/utils.ts [hunk 2]
```diff
@@ -9,19 +9,24 @@ export const isString = (something: unknown): something is string => typeof some
 export const isRecord = (something: unknown): something is Record<string, any> =>
   typeof something === 'object' && something !== null
 
+interface EqualsOption {
+  checkExtraKeys?: boolean
+  deep?: boolean
+}
+
 export const objectEquals = <T extends Record<string, any>>(
```
Comment (jodator) [inline packages/ui/src/common/utils.ts:17]: This one would require a test. There's a lot going on and a test could clear up how it should be used.

**Review by jodator — CHANGES_REQUESTED** [review]
The component works very well :) I'd like to ask you only to add a test and move the z-index declaration to constant so we could have better z-index management.

**Pull request merged**